analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -143,10 +144,21 @@ class GoMod(
             .requireSuccess()
             .stderr
             .lines()
-            .filter { it.startsWith("# ") }
-            .map {
-                val parts = it.removePrefix("# ").split(" ", limit = 2)
-                Identifier(managerName, "", parts[0], parts[1])
+            .mapNotNull { line ->
+                // The package can be replaced using GoMod's replace directive. This should only take the used package
+                // into account, see: https://golang.org/doc/modules/gomod-ref#replace.
+                val replaceIndicator = " => "
+
+                val withoutPrefix = line.withoutPrefix("# ")
+                val moduleVersionLine = if (line.contains(replaceIndicator)) {
+                    withoutPrefix?.substring(line.indexOf(replaceIndicator) + replaceIndicator.length)
+                } else {
+                    withoutPrefix
+                }
```

Comment:
Can we simply do:

                val moduleVersionLine = line.withoutPrefix("# ")?.let {
                    line.split(replaceIndicator).last()
                }